components/admin_console/data_grid/data_grid.scss

```diff
@@ -83,14 +89,19 @@
 
         .DataGrid_row {
             min-height: 50px;
-
+            
             &:nth-child(odd) {
-                background: $bg--gray;
+                background-color: $bg--gray;
             }
+
+            &:hover {
+                background-color: lighten($primary-color, 45%);
+                border-left: 2px solid $primary-color;
+            }
```

Comment:
Adding this border 2px causes the width of the element to increase so on hover it looks like the row is shifting slightly to the right as seen below
![Kapture 2020-06-12 at 14 30 48](https://user-images.githubusercontent.com/3207297/84535199-526ca300-acb9-11ea-81e7-712a5f96bf1c.gif)

In order to fix that you should add a transparent border of the same size to `.DataGrid_row ` above
something like 
`border-left: 2px solid transparent;` should be good